packages/now-cli/src/util/input/edit-project-settings.ts

```diff
@@ -4,32 +4,30 @@ import chalk from 'chalk';
 import { Output } from '../output';
 import { Framework } from '@vercel/frameworks';
 import { isSettingValue } from '../is-setting-value';
+import { ProjectSettings } from '../../types';
 
-export interface ProjectSettings {
+export interface PartialSettings {
```

Comment:
Could we rename this to `PartialProjectSettings`, I think just `PartialSettings` sounds a bit too generic.

Reply:
Renamed 👍